arangod/Transaction/Manager.h

```diff
@@ -62,6 +67,30 @@ class Manager final {
 
  public:
   typedef std::function<void(TRI_voc_tid_t, TransactionData const*)> TrxCallback;
+  
+  enum class MetaType : uint8_t {
```

Comment:
I think instead of making this public, how about just passing in the TransactionState* ?